### PR TITLE
Modify header and footer

### PR DESCRIPTION
**Pull request by jyx11011**
Screenshot 2020-07-13 at 6 02 13 PM

image

### CODE REVIEW

File: src/main/webapp/WEB-INF/components/footer.html [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
-      <div class="col-lg-4 mb-5 mb-lg-0">
-        <h4 class="text-uppercase mb-4">Around the Web</h4>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-facebook-f"></i></a>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-twitter"></i></a>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-linkedin-in"></i></a>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-dribbble"></i></a>
+      <div class="col-lg-8 mb-5 mb-lg-0">
+        <h4 class="text-uppercase mb-4">About us</h4>
+        <div class="row">
+          <div class="col-lg-4">
```
Comment (limli) [inline src/main/webapp/WEB-INF/components/footer.html]:
```suggestion
          <div class="col-lg-4" mb-3 mb-lg-0>
```
So it doesn't look to cramped on small screens. Same for the other 2 divs.

File: src/main/java/com/google/step/servlets/CommentGetPostServlet.java [hunk 2]
```diff
@@ -33,7 +33,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
       return;
     }
-    List<Comment> comments = manager.getComments(dealId);
+    List<Comment> comments = manager.getCommentsForDeal(dealId);
```
Comment (limli) [inline src/main/java/com/google/step/servlets/CommentGetPostServlet.java]: I think this should be in another PR

File: src/main/webapp/js/util.js [hunk 3]
```diff
@@ -22,17 +22,23 @@ function configureHeaderLoginStatus() {
...
+          const logoutLink = document.getElementById('logout-link');
+          logoutLink.href = loginStatus.logoutUrl;
         } else {
-          loginStatusLink.href = loginStatus.loginUrl;
-          loginStatusLink.innerText = 'Login';
-          headerUsername.innerText = '';
+          loginLink.hidden = false;
+          usernameDropdown.hidden = true;
+          loginLink.href = loginStatus.loginUrl;
+          headerUsername.hidden = true;
```
Comment (limli) [inline src/main/webapp/js/util.js:39]: `headerUsername` is not defined here

File: src/main/java/com/google/step/servlets/CommentGetPostServlet.java (hunk 2 above)
Comment (jyx11011) [reply src/main/java/com/google/step/servlets/CommentGetPostServlet.java]: ok will wait for #50

**Review by limli — DISMISSED** [review]

**Review by anuragshirolkar — APPROVED** [review]
